docker-compose.yml

```diff
@@ -240,6 +240,19 @@ services:
       - "1994:1994"
     environment:
       - NODE_ENV=development
+
```

Comment:
let's set the release in a way that uses ironwood in required services like line 228 `image: edxops/devpi:${OPENEDX_RELEASE:-latest}`